SECURITY.md

```diff
@@ -30,10 +39,14 @@ Confio uses the following disclosure process for the various CosmWasm-related re
 
 - Once a security report is received, the core development team works to verify the issue.
 - Patches are prepared for eligible releases in private repositories.
-- We notify the community that a security release is coming, to give users time to prepare their systems for the update. Notifications can include Discord messages, tweets, and emails to partners and validators. Please also see [CosmWasm/advisories](https://github.com/CosmWasm/advisories) if you want to receive notifications.
-- No less than 24 hours following this notification, the fixes are applied publicly and new releases are issued.
+- We notify the community that a security release is coming, to give users time to prepare their systems for the update.
+  - The notification contains the release date and time (72-24 hours after the notification).
+  - The notification contains as little information as possible to make it hard for attackers to guess where to search. However, depending on the situation it may contain additional information such as whether or not the patch will be consensus breaking or chains with permissioned CosmWasm integration are equally affected as permissionless chains.
```

Comment:
```suggestion
  - The notification contains as little information as possible, to make it hard for attackers to guess where to search. However, depending on the situation it may contain additional information such as whether the patch will be consensus breaking, or if chains with permissioned CosmWasm integration are equally affected as permissionless chains.
```